db/schema.rb

```diff
@@ -79,6 +79,16 @@
     t.index ["document_id"], name: "index_images_on_document_id"
   end
 
+  create_table "timeline_entries", force: :cascade do |t|
+    t.string "entry_type", null: false
+    t.bigint "document_id", null: false
+    t.bigint "user_id", null: false
```

Comment:
Be good to keep this allowable to be nil so that it doesn't cause too many pain points with regards to importing data from Whitehall. Our other places where we have a user_id column are nullable.